Dockerfile

```diff
@@ -31,6 +31,7 @@ FROM amazon/aws-lambda-python:$PY_VERSION as build-stage
 # For a list of pre-defined annotation keys and value types see:
 # https://github.com/opencontainers/image-spec/blob/master/annotations.md
 ###
+LABEL dummy="dummy"
```

Comment:
Can this be deleted?
```suggestion
```

Reply:
[It _is_ deleted in the branch](https://github.com/cisagov/findings-data-import-lambda/blob/93ff85ab92439a9dec03a75a65bedc30ceef6b42/Dockerfile#L31-L35), but the PR isn't updating for some reason.

Reply:
Ah, I see.  GitHub weirdness strikes again!